test/junit/scala/collection/immutable/ArraySeqTest.scala

```diff
@@ -123,4 +123,60 @@ class ArraySeqTest {
     assert(ArraySeq.from(as) eq as)
     assert(ArraySeq(as: _*) eq as)
   }
+
+  private def assertConcat[A](lhs: ArraySeq[A], rhs: ArraySeq[A], expect: ArraySeq[A]): Array[_] = {
+    val appended = lhs ++ rhs
+    val prepended = lhs ++: rhs
+    assertEquals(appended, expect)
+    assertEquals(prepended, expect)
+    assertEquals(appended.unsafeArray.getClass, prepended.unsafeArray.getClass)
+    appended.unsafeArray
+  }
+
+  @Test
+  def concatPrimative(): Unit = {
```

Comment:
`primitive`. I had to look up why. "primitivus" vs "primarius", "primitive" vs "primary". Are they synonymous? There is the sense of primary as "first in rank", primas is archbishop, primate. "prime" has both connotations in prime number and prime cut of beef. Webster's suggests noun and adjective prime have slightly differing histories. Although primitive also means first, underived, elemental, etc, I don't think it is ever valorized, and conversely primary never has the pejorative of sense of merely primitive.

Reply:
Hahaha thanks @som-snytt ! Your reciprocity of nitpicking PRs came to late to save me! It will forever go down in Scala history that I'm terrible at spelling "primitive". And no I'm not being hard on myself, I honestly make this mistake *all the time*. There's a few words like that that for some reason I can just never seem to remember how to spell. :laughing:

Reply:
Me too, sometimes I remember my etymological mnemonic, but then I forget how it applies. In this case, I'm willing to put primative on equal footing with its and it's. There may be a good pun that turns on privative, or possibly a limerick using privilege and privet hedge.

I just remembered that on today's jog, I came up with my t-shirt for a job search:
`"10x, anyone?"`

Reply:
I'm actually good with its & it's. The way I remember it is that it's a special case which is enough for me to quickly run through the logic that the `'s` for possession doesn't apply in this case.

Shirt sounds great! Maybe with `yeah, I think of stuff on jogs. I'm pretty fit. no big deal.` on the back :D